charts/argo-rollouts/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: v1.7.0
 description: A Helm chart for Argo Rollouts
 name: argo-rollouts
-version: 2.36.2
+version: 2.36.3
```

Comment:
changelog is not updated, please update like this.

```yml
    - kind: fixed
      description: Fix subscriptions into notifications-configmap

```

Reply:
Updated.